features/enlinkd/org.opennms.features.enlinkd.generator/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <parent>
+    <groupId>org.opennms.features</groupId>
+    <artifactId>org.opennms.features.enlinkd</artifactId>
+    <version>24.0.0-SNAPSHOT</version>
+  </parent>
+  <properties>
+    <bundle.symbolicName>org.opennms.features.enlinkd.generator</bundle.symbolicName>
+    <bundle.namespace>org.opennms.features.enlinkd.generator</bundle.namespace>
+  </properties>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.opennms.features.enlinkd</groupId>
+  <artifactId>org.opennms.features.enlinkd.generator</artifactId>
+  <name>OpenNMS :: Features :: Enlinkd :: Topology Generator</name>
+  <packaging>bundle</packaging>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
+            <Karaf-Commands>*</Karaf-Commands>
+          </instructions>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
+    <!--
+     | uncomment to add all imported (non-local) bundles to your compilation classpath
+    <dependency>
+      <type>pom</type>
+      <groupId>${parent.groupId}</groupId>
+      <artifactId>provision</artifactId>
+      <optional>true</optional>
+    </dependency>
+    -->
+    <dependency>
```

Comment:
This dependency should not be necessary.

Reply:
fixed